dmn-engine/src/test/scala/org/camunda/dmn/AuditLogTest.scala

```diff
@@ -47,6 +53,69 @@ class AuditLogTest extends AnyFlatSpec with Matchers with DecisionTest {
     auditLog.requiredEntries.size should be(0)
   }
 
+  it should "contains the result of a decision table if there is none" in {
```

Comment:
Make the description a bit more clear.

```suggestion
  it should "contains the result of a decision table if no rule matched" in {
```